src/main/ml-modules/root/com.marklogic.smart-mastering/matcher-impl/matcher-impl.xqy

```diff
@@ -500,3 +519,17 @@ declare function match-impl:results-to-json($results-xml)
     )/node()
   else ()
 };
+
+declare function match-impl:instance-query-wrapper(
+  $query as cts:query,
+  $is-json as xs:boolean
+) {
+  if ($is-json) then
+    if (fn:exists($const:JSON-INSTANCE)) then
```

Comment:
Related to above, under what circumstances would `$const:JSON-INSTANCE` fail to exist?

Reply:
I added the condition should someone want to override the constants to not restrict to an instance body.

Reply:
okay